Source/Csla/Server/DataPortal.cs

```diff
@@ -127,7 +127,7 @@ private Reflection.ServiceProviderMethodCaller ServiceProviderMethodCaller
 
         AuthorizeRequest(new AuthorizeRequest(objectType, criteria, DataPortalOperations.Create));
 
-        Initialize(new InterceptArgs { ObjectType = objectType, Parameter = criteria, Operation = DataPortalOperations.Create, IsSync = isSync });
+        await Initialize(new InterceptArgs { ObjectType = objectType, Parameter = criteria, Operation = DataPortalOperations.Create, IsSync = isSync });
```

Comment:
If we're going to have a breaking change anyway, let's change this to `InitializeAsync` to be consistent with other CSLA code.

Reply:
> If we're going to have a breaking change anyway, let's change this to `InitializeAsync` to be consistent with other CSLA code.

Only for the internal method or all changed methods?

Reply:
I think the interface (public) changed to return a `Task` right? So that method name should change, and then presumably all the non-public ones too?

Reply:
The public and some internal methods needed to change, too. So I'll just change all touched methods and add the Async suffix.

Reply:
Done. Found some missed Tests which did not await the change which is now fixed, too.